digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala

```diff
@@ -1589,6 +1589,83 @@ object DataFixture {
     println("\n")
   }
 
+
+  private def isValidUnknownSpeedLimit(roadLink: RoadLink): Boolean = {
```

Comment:
This method (isValidUnknownSpeedLimit) is duplicated.
Was created here and on SpeedLimitFiller.scala file, try to put him in a place who can be used by both files.

Reply:
I would go with @oskarmalmstrom idea.

> We could put this inside the RoadLink case class in case we need to use it in some other place. (like it is done with the `def isCarTrafficRoad`)